vehicle_controller:
  ros__parameters:
    manual_override_active: False
    manual_steer: 0.0
    manual_throttle: 0.0

    loop_sleep_time: 0.2

velocity_controller:
  ros__parameters:
    pid_p: 0.6
    pid_i: 0.00076
    pid_d: 0.63
    fixed_speed: 0.0
    fixed_speed_active: False

pure_pursuit_controller:
  ros__parameters:
    k_lad: 0.85
    min_la_distance: 3.0
    max_la_distance: 25.0
    k_pub: 1.0
